docs/examples.rst

```diff
@@ -0,0 +1,34 @@
+.. _examples_label:
+
+~~~~~~~~~~~~~~~
+Examples
+~~~~~~~~~~~~~~~
+
+Example 1
+=====
+The model consists of two nodes, one for Brandenburg and one for Berlin.
+
+Modelled components are 
+
+- renewable energies (wind, photovoltaic, biomass)
+- conventional power plants (gas, oil, others)
+
+Data for capacities of wind, photovoltaic and biomass as well as electricity demand have been taken from
+the grid development plan (`Netzentwicklungsplan <https://www.netzentwicklungsplan.de/sites/default/files/paragraphs-files/NEP_2035_V2021_1_Entwurf_Teil1.pdf>`_, p. 41 ff.).
+Data for efficiency and costs (specific annuity, fuel costs) are based on different sources as well as own assumptions and can be adjusted.
+
+
+All electricity generating components can be expanded. 
+Additionally, it can be invested into grid expansion and battery storage.
+
+Three scenarios have been built based on the grid development plan:
+
+- simple_model: corresponds to scenario A 2035 of the grid development plan
```

Comment:
We should think about names that are more telling.

Reply:
@c-moeller: I wonder how close the correspondence to NEP is? Would it justify to call the scenarios by A-2035 etc.? Or would it be more honest to say that the scenarios have been setup with a loose correspondence to the scenarios in NEP by reflecting higher wind capacities and less ch4-gt in 2040? This should be consistent with the naming and explanations at OEP. There, the scenarios seem to be called scenario 1, scenario 2, scenario 3. Is that fixed, @christian-rli? In that case we should rename the scenarios here accordingly.

Reply:
The naming is not fixed. In fact, we thought it would be useful to give more descriptive names, so feel free to rename wildly and let us know the result :)

Reply:
Good! Can you give some input here along the questions posed above or shall we discuss this, @c-moeller?

Reply:
Does that help? Or do you need more information concerning the scenarios?

Reply:
We came up with this:
* basic
* more_re
* more_re_less_fossil